.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+env:
+  GOPROXY: https://proxy.golang.org/
+  PROTOCOL_VERSION: "5.0"
+  PROTOCOL_VERSIONS: "5.0"
+  SIGNER: interim_signing_subkey_7685B676
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Read go version
+        id: go-version
+        run: |
+          content=`cat ./.go-version`
+          echo "::set-output name=content::$content"
+      - terraform-provider-release:
+        name: 'Terraform Provider Release'
+        needs: go-version
+        uses: bflad/ghaction-terraform-provider-release/.github/workflows/hashicorp.yml@main
+        secrets:
+          hc-releases-aws-access-key-id: '${{ secrets.TF_PROVIDER_RELEASE_AWS_ACCESS_KEY_ID }}'
+          hc-releases-aws-secret-access-key: '${{ secrets.TF_PROVIDER_RELEASE_AWS_SECRET_ACCESS_KEY }}'
+          hc-releases-aws-role-arn: '${{ secrets.TF_PROVIDER_RELEASE_AWS_ROLE_ARN }}'
+          hc-releases-fastly-api-token: '${{ secrets.FASTLY_STATIC_PURGE_TOKEN }}'
+          hc-releases-github-token: '${{ secrets.HC_RELEASES_TOKEN }}'
+          hc-releases-terraform-registry-sync-token: '${{ secrets.TF_PROVIDER_RELEASE_TERRAFORM_REGISTRY_SYNC_TOKEN }}'
+          setup-signore-github-token: '${{ secrets.SETUP_SIGNORE_GITHUB_TOKEN }}'
+          signore-client-id: '${{ secrets.SIGNORE_CLIENT_ID }}'
+          signore-client-secret: '${{ secrets.SIGNORE_CLIENT_SECRET }}'
+        with:
+          # Another job, whatever appropriate for the project, could generate the release notes
+          # goreleaser-release-args: '--release-notes /path/to/file'
+          setup-go-version: '${{ needs.go-version.outputs.version }}'
+          terraform-registry-manifest-source: 'terraform-registry-manifest.json'
```

Comment:
Let's also ensure there's a manifest file in the repository as part of this, e.g. create a `terraform-registry-manifest.json` file at the root of the repository with contents:

```json
{
    "version": 1,
    "metadata": {
        "protocol_versions": ["5.0"]
    }
}
```

The reusable workflow will take care of renaming the file into the naming convention the Terraform Registry expects in the release assets (which is why it looks different in the goreleaser suggestions below). 👍